code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -613,13 +649,27 @@
 					message = "buzzes."
 				playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 50, 0)
 				m_type = 1
+		
+		if("vomit")
+			if (isipc(src))

```

Comment:
I'm pretty sure there's a, "hasmouth()" proc or something like that. Use that.

Reply:
check_has_mouth() is the call you want